synapse/storage/data_stores/main/schema/delta/56/redaction_censor.sql

```diff
@@ -14,4 +14,6 @@
  */
 
 ALTER TABLE redactions ADD COLUMN have_censored BOOL NOT NULL DEFAULT false;
-CREATE INDEX redactions_have_censored ON redactions(event_id) WHERE not have_censored;
+
+INSERT INTO background_updates (update_name, progress_json) VALUES
```

Comment:
I'm a bit surprised we need this at all tbh

Reply:
As in, the index?

Reply:
ja